lib/rules/media-feature-range-notation/index.mjs

```diff
@@ -80,20 +80,26 @@ const rule = (primary, _secondaryOptions, context) => {
 									-1,
 									{ value: unprefixedMediaFeature },
 								]),
-								node.name.before,
-								node.name.after.length > 0
-									? node.name.after
+								entry.node.name.before,
+								entry.node.name.after.length > 0
+									? entry.node.name.after
 									: [[TokenType.Whitespace, ' ', -1, -1, undefined]],
 							),
 							[operator, [TokenType.Delim, '=', -1, -1, { value: '=' }]],
-							node.value,
+							entry.node.value,
 						);
 
-						return;
-					}
+						const expectedMediaQueryList = mediaQueryList.map((mq) => mq.toString()).join(',');
 
-					const [startIndex, endIndex] = sourceIndices(node);
+						if (expectedMediaQueryList === atRule.params) return;
+
+						atRule.params = expectedMediaQueryList;
+					};
 
+					const hasFix =
+						primary === 'context' && isMediaFeaturePlain(node) && isMediaFeature(parent);
+					const fix = hasFix ? contextFixer({ node, parent }) : undefined;
```

Comment:
[suggest] Super minor. `mayFix` (like a variable in `report()`) seems a bit more accurate to me. It may be just an issue of preference, though.

```suggestion
					const mayFix =
						primary === 'context' && isMediaFeaturePlain(node) && isMediaFeature(parent);
					const fix = mayFix ? contextFixer({ node, parent }) : undefined;
```